rotkehlchen/globaldb/handler.py

```diff
@@ -237,6 +237,20 @@ def cleanup(self) -> None:
         if self._packaged_db_conn is not None:
             self._packaged_db_conn.close()
 
+    @staticmethod
+    def _wal_checkpoint(conn: DBConnection) -> None:
+        """
+        Commit wal file. If database is locked we ignore it. This is needed when attacking
+        databases and we want to have up to date information
+        """
```

Comment:
Also can you add a link to the docs here:  https://www.sqlite.org/pragma.html#pragma_wal_checkpoint ?